blogpost.md

```diff
@@ -1,3 +1,7 @@
+With the recent [release of .NET 7](https://devblogs.microsoft.com/dotnet/announcing-dotnet-7/), we'd like to introduce some interesting changes and additions done in the networking space. This blog post talks about .NET 7 changes in [HTTP space](#http), [new QUIC APIs](#quic), [networking security](#security) and [WebSockets](#websockets).
+
+You can find similar blog posts for previous releases here: [.NET 6 Networking Improvements](https://devblogs.microsoft.com/dotnet/dotnet-6-networking-improvements/), [.NET 5 Networking Improvements](https://devblogs.microsoft.com/dotnet/net-5-new-networking-improvements/).
```

Comment:
I think this would look better as a part of outro 🤔

Reply:
Done